lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -528,6 +529,15 @@ void OpenCLFunction::execute() {
                                      : ElemKind::FloatTy;
     std::string kernelName = getKernelName(I.getKindName(), elemTy);
 
+    //  Check if the instruction is quantized.
+    bool isQuantized = false;
+    if (I.getNumOperands() &&
+        I.getOperand(0).first->getType()->isQuantizedType()) {
```

Comment:
checking "isQuantized" should be enough, you could remove "if" also and do this in one statement.